lib/active_remote/association.rb

```diff
@@ -135,6 +138,7 @@ def validate_scoped_attributes(associated_class, object_class, options)
       def create_setter_method(associated_klass, options={})
         writer_method = "#{associated_klass}=".to_sym
         define_method(writer_method) do |new_value|
+          raise "New value must be an array" if options[:has_many] == true && new_value.class != Array
```

Comment:
Do you know how ActiveRecord behaves when you attempt to do this? I think it it would make sense to do mimic what it's doing.